#!/usr/bin/env python
PACKAGE = "kinect_scan"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("DistanceThreshold_Ransac",    double_t,    0, "Minimum distance between two points",  0.5, 0,   3)

gen.add("SearchRadius",   double_t,   0, "Ransac filter",   0.5, 0,   2)

gen.add("seuil",   double_t,   0, "Threshold to keep a pt cloud",   0.05, 0,   1.0)

#gen.add("input_cloud",   str_t,   0, "Input cloud",   "/cloud_file_segmentation/pointCloud_segmented2")

#gen.add("apply_change",   bool_t,   0, "Save the filtered cloud",   False)



exit(gen.generate(PACKAGE, "kinect_scan", "ransac_segmentation"))


